reddit_decider/__init__.py

```diff
@@ -135,6 +139,7 @@ def to_event_dict(self) -> Dict:
             "device_id": self._device_id,
             "origin_service": self._origin_service,
             "cookie_created_timestamp": self._cookie_created_timestamp,
+            "loid_created_timestamp": self._loid_created_timestamp,
```

Comment:
similar here, there's no top level field in v2 event schema for loid (or [any field](https://github.snooguts.net/reddit/data-schemas/search?q=loid) for loid in v2 events) so let's get rid of it here as well